lldb/packages/Python/lldbsuite/test/make/Makefile.rules

```diff
@@ -103,6 +103,17 @@ else
 endif
 endif
 
+#----------------------------------------------------------------------
+# Override CC and CXX when building a hip application.
```

Comment:
I have realized that I need to change this file as well for NVIDIA. I tried to pass the path of my compiler to the python test file when running with lldb-dotest but it didn't work. It seems that the path is only correctly passed when running check-lldb.